scripts/debug_course_publisher.py

```diff
@@ -7,33 +7,50 @@
 from geometry_msgs.msg import TransformStamped
 
 from mrover.msg import Course, Waypoint
+import mrover.srv
 
 
-def send_waypoint(frame_id: str, x: float, y: float, fid_id: int = -1) -> Waypoint:
+def add_waypoint(frame_id: str, x: float, y: float, fid_id: int = -1) -> Waypoint:
+    w = Waypoint()
+    w.fiducial_id = fid_id
+    w.tf_id = frame_id
+    return w
+
+
+def get_waypoint_tf(frame_id: str, x: float, y: float, fid_id: int = -1):
```

Comment:
Maybe just me, but potentially a comment describing what happens here? Same goes for similar functions unless either trivial enough or already explained elsewhere